### DISCUSSION TITLE
Event content type

### DISCUSSION THREAD

**Discussion by lauggh** [worldiaday/website#23] [Content, Design, and IA]
Event content type is used for events that have at min: 
- event name
- datetime, 
- event organizers

![image](https://user-images.githubusercontent.com/603924/122515447-4ef1b300-cfc2-11eb-8805-f1e1106e811f.png)

Fields identified so far: 
- Event title
- Event description
- Event start datetime
- Event end datetime
- Livestream
- Event status
- Speakers
- Event format
- How to join
- Slug
- Local organizer
- Volunteer team
- Event sponsor
- Event program
- Session title
- Session duration

Events have local, global, regional reach, such that it does not have to have a location associated with it. This is different from the current model in Drupal where the event name is formulated based on the year/theme and the location name. 

Events have people references that have contextual roles.  For example, an event organizer for one year may become a speaker another year. Thus their role should be tied to the event that they're currently associated with, rather than tied to the Person content type. 

This might mean that a person reference has a Role attribute. Might also be interesting if this role attribute can be pulled into a Person content such that we can list out all the ways that a person contributes as part of World IA Day... as a organizer, speaker, author, regional director, GEP, volunteer, etc. 

**Events are made up of speakers.** 
The assumption is that sometimes events have identified speakers, but not have titles for talks/sessions just yet. 

**Events are made up of sessions.** 
Sessions can be networking, welcome, closing, lunch, and breaks, for instance. Sessions can also be talks, workshops. Sessions that have recordings can be linked to a video asset, as well as presentations, transcripts, translations. 

**Events have sponsors and partners.** 
Sponsors and partners may be grouped based on each event/location's sponsorship packages and/or patronage level. Expectation that each sponsor and partner should have attributes: 
- name
- tracking URL
- logo
- group

Sponsors and partners for an event can also be ordered in a specific way...

**Comment by lauggh** [comment]
Suggested by @souvikdg, this should be updated to include:  
- event location, image/banner/logo, event contact
- See structured data guide: https://developers.google.com/search/docs/data-types/event

Might also refer to an event program as a content type that can link to sessions. Program might have broken/grouped into "Days"

Should also include eventStatus -- we added some of this already, but should see the standards for indicating online, in-person, and hybrid events and how those are communicated. 

@jessishakarian As you're going through the content type for Event, could you make sure to include these?

**Reply by jessishakarian** [reply]
Will do!

**Reply by jessishakarian** [reply]
Made a session type for in-person, virtual, hybrid since it's going to vary from session to session. Connected that to event format.
